front_end/README.md

```diff
@@ -1,21 +1,13 @@
-# HTML/CSS/JS Project
+# Ember Project
 
-We believe that a good front end should be built using semantic markup and well-structured CSS. To that end, it's important that our front end developers demonstrate the ability to convert a mocked-up page image into well-written code. The goal of this excercise is to take the mockups provided and build an HTML/CSS representation of them.
+We've put your soft skills to the test, now we want to see some dope code! For this part of the interview, we would like you to create a small Ember.js app using this [API](https://reqres.in/). We would like for you to use ember-data to talk to this API, but if that turns out too complex, feel free to use a simpler library like [ember-ajax](https://github.com/ember-cli/ember-ajax).
 
 If any special requirements have been conveyed to you from your interview
-team, they supercede the following.
+team, they supersede the following.
 
 ## Requirements
 
+- [ ] Create a paginated list of users.
+- [ ] Each user should be able to toggle a side sliding panel/modal that is populated with that user's data.
+- [ ] Style it to the best of your ability using sass.
 - [ ] Do not use a front end framework such as bootstrap.
```

Comment:
Should we clarify that it's fine to use a utility framework like tailwind or neat?

Reply:
That's fine if you think it's important. Should I mention making it as responsive as possible too?

Reply:
Yeah, that makes sense!